synapse/cortex.py

```diff
@@ -87,6 +87,11 @@ def choptag(tag):
 if __name__ == '__main__':  # pragma: no cover
```

Comment:
+1 on removing code.interact.  If we move this into using a main() function, it'll become testable & we can drop the pragma no cover statement.